### ISSUE TITLE
Can I deploy it with Heroku?

### ISSUE THREAD

**Issue by jxpin84** [CompVis/stable-diffusion#18]

**Comment by breadbrowser** [comment]
Heroku only have cpus

**Comment by mlaprise** [comment]
With Hectiq Lab (https://lab.hectiq.ai), It's possible to deploy Streamlit apps on GPU (Nvidia T4).

**Comment by breadbrowser** [comment]
> With Hectiq Lab (https://lab.hectiq.ai), It's possible to deploy Streamlit apps on GPU (Nvidia T4).

that site is harder to learn then blender

**Comment by breadbrowser** [comment]
> With Hectiq Lab (https://lab.hectiq.ai), It's possible to deploy Streamlit apps on GPU (Nvidia T4).

it costs money

**Comment by mlaprise** [comment]
yeah sorry about the confusion. Deploying a GPU always cost money (except on Colab). There is a free tier CPU only but I'm not sure it's a viable solution for this model

**Comment by breadbrowser** [comment]
> yeah sorry about the confusion. Deploying a GPU always cost money (except on Colab). There is a free tier CPU only but I'm not sure it's a viable solution for this model

then explain kaggle.com

**Comment by breadbrowser** [comment]
the demo is here https://huggingface.co/spaces/stabilityai/stable-diffusion
